src/js/components/ConnectionModals/ViewConnectionModal.tsx

```diff
@@ -25,6 +30,11 @@ const StyledFooter = styled.footer`
   button {
     margin-left: 12px;
   }
+
+  a {
+    flex: 1;
+    color: var(--havelock);
```

Comment:
```suggestion
    margin-right: auto;
    color: var(--red);
```

flex: 1 will make all the space to the right of the "Remove" text also clickable. margin-right: auto achieves the same positioning without making that region part of the link.


I think red looks good here:

<img width="451" alt="image" src="https://user-images.githubusercontent.com/3460638/99604433-5617fa80-29ba-11eb-9a7b-5e01dd67b1a1.png">